hector.md

```diff
@@ -0,0 +1,61 @@
+
+![Hector Carbon Cycle diagram](gcam-figs/hector_box_model.png)<br/>
+Figure 1: Representation of Hector’s carbon cycle, land, atmosphere, and ocean. The atmosphere consists of one well-mixed box. The ocean consists of four boxes, with advection and water mass exchange simulating thermohaline circulation. At steady state, the high-latitude surface ocean takes up carbon from the atmosphere, while the low-latitude surface ocean off-gases carbon to the atmosphere. The land consists of a user-defined number of biomes or regions for vegetation, detritus and soil. At steady state the vegetation takes up carbon from the atmosphere while the detritus and soil release carbon back into the atmosphere. The earth pool is continually debited with each time step to act as a mass balance check on the carbon system. 
+
+
+For more information on Hector please see: 
+[Hector - github repository](http://github.com/JGCRI/Hector)
+[Hartin et al. 2015](http://www.geosci-model-dev.net/8/939/2015/)
+[Hartin et al. 2016](http://www.biogeosciences.net/13/4329/2016/bg-13-4329-2016.html)
+
+##GCAM-Hector interactions
+Currently the GCAM sectors interact with Hector via their emissions.  At every time step, emissions from GCAM are passed to Hector. Hector takes in emissions, converts to concentrations where necessary, and calculates the radiative forcing, as well as the resulting climate variables (e.g., temperature, carbon-fluxes, etc.)  
+
+| Emission| Sector  | Notes |
+| ------- |:-------:| :------: |
+| CO<sub>2</sub>     | AgLU, Energy  | |
+| CH<sub>4</sub>     | AgLU    | |
+| N<sub>2</sub>O 	  | AgLU    | |
+| NH<sub>4</sub>     | AgLU    | not included in Hector |
+| SO<sub>2</sub>    | AgLU    | |
+| CO 	  | AgLU    |         |
+| BC      | AgLU, Energy    | |
+| OC      | AgLU, Energy    ||
+| NO<sub>x</sub> 	  | AgLU    | |
+Table 1: Emissions and sources from each sector passed to Hector. 
+
+##Hector Outputs
+At every time step Hector calculates and outputs key climate variables.  
+#####Atmosphere
+Global mean temperature change
+Radiative forcing of all emissions
+Atmospheric CO<sub>2</sub> concentrations
+#####Land
+Air-land carbon fluxes
+NPP - net primary production
+RH - heterotrophic respiration
+Carbon pools (vegetation, detritus, soil)	
+#####Ocean
+Air-sea carbon fluxes
+Carbon pools (high and low latitude surface, intermediate and deep)
+Carbonate system (DIC, pCO<sub>2 </sub>, CO<sub>3</sub><sup>2-</sup>, pH, aragonite and calcite saturations)
+surface ocean temperature
+oceanic heat flux
+
+##Building Hector + GCAM
+For instructions on how to build GCAM with Hector click [here](https://confluence.pnnl.gov/confluence/pages/viewpage.action?pageId=63278699)
```

Comment:
You can't include a link to a PNNL Confluence page in these docs because most users will not be from PNNL and therefore won't be able to access our Confluence site.  Probably you are going to need to repeat the information from the Confluence document in this section.